pensieve/pensieve.py

```diff
@@ -146,10 +146,22 @@ def paragraphs(self):
         if not self._paragraphs:
             print('Generating paragraphs for doc '+str(self.id))
             self._paragraphs = []
-            for i, p in enumerate(self.text.split('\n')):
-                if len(p.split(' ')) < 30:
+            list = self.text.split('\n')
```

Comment:
You want to avoid naming variables after Python built-in functions (like `list`). If you want to use this built-in later in the code, you'll get unexpected behavior because the function has been remapped in the namespace to a variable. This variable should be named something like `line_list` to avoid this problem